meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -534,6 +543,72 @@ PREFERRED_PROVIDER_u-boot-default-script:stm32mp1common = "u-boot-base-scr"
 PREFERRED_PROVIDER_u-boot-default-script:sota:stm32mp1common = "u-boot-ostree-scr-fit"
 # Remove standard ST BSP from SDK
 TOOLCHAIN_HOST_TASK:remove:stm32mp1common = "${ST_TOOLS_FOR_SDK} ${ST_DEPENDENCIES_BUILD_FOR_SDK}"
+STM32_BOOTIMAGE_SUFFIX ??= ""
+MFGTOOL_FLASH_IMAGE ?= "${IMAGE_BASENAME}-${MACHINE}"
+LMP_FLASHLAYOUT_IMAGE = "${MFGTOOL_FLASH_IMAGE}.ext4"
+LMP_FLASHLAYOUT_IMAGE:sota = "${MFGTOOL_FLASH_IMAGE}.ota-ext4"
+FLASHLAYOUT_BOOTSCHEME_LABELS = "lmp"
```

Comment:
Please add a comment here saying where these variables are used (stm layer, flashlayout class).

Also please add something related to this in the commit message, the body is empty (just to avoid patches with subject only information).